packages/create-cloudflare/src/common.ts

```diff
@@ -325,10 +327,31 @@ export const gitCommit = async (ctx: PagesGeneratorContext) => {
 	if (!(await isGitInstalled()) || !(await isInsideGitRepo(ctx.project.path)))
 		return;
 
+	const git = simpleGit();
+
+	await git.add(".");
+
+	const gitStatus = await simpleGit().status();
+
+	if (gitStatus.not_added.length > 0) {
```

Comment:
PS: we could reuse `simpleGit` for the other various operations but in various places, like the commit below it would require some refactoring so that its result can be displayed nicely

Reply:
I'm cool with giving this a shot 👍